LayoutTests/fast/scrolling/scrollbar-gutter-stable-rtl-viewport.html

```
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<style>
+iframe {
+    width: 200px;
+    height: 200px;
+    border: none;
+}
+</style>
+<p>The scrollbar gutter should be on the left side (RTL).</p>
+<iframe src="data:text/html,<html dir='rtl' style='scrollbar-gutter:stable;margin:0'><body style='margin:0'><div style='height:10px;background:green'></div></body></html>"></iframe>
```

HTMLHint (1.9.2)

[warning] A <iframe> element must have an accessible name.

(frame-title-require)